cli/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "zip-cli"
+version = "0.0.0"
+authors = [
+    "Danny McClanahan <dmcC2@hypnicjerk.ai>",
+]
+license = "MIT"
+repository = "https://github.com/zip-rs/zip2.git"
+keywords = ["zip", "archive", "compression", "cli"]
+categories = ["command-line-utilities", "compression", "filesystem", "development-tools::build-utils"]
+rust-version = "1.73.0"
```

Comment:
Increase this in order to update `clap`.

Reply:
Done!